shared/global-errors/index.desktop.js

```diff
@@ -65,35 +64,35 @@ class GlobalError extends Component<void, Props, State> {
     if (nextProps.error !== this.props.error) {
       this.props.setTimeout(() => {
         this.setState({
-          cachedSummary: this._summaryForError(nextProps.error),
           cachedDetails: this._detailsForError(nextProps.error),
+          cachedSummary: this._summaryForError(nextProps.error),
         })
-      }, nextProps.error ? 0 : 3000) // if its set, do it immediately, if its cleared set it in a bit
+      }, nextProps.error ? 0 : 7000) // if its set, do it immediately, if its cleared set it in a bit
```

Comment:
increase the timeout some. several complaints about not being able to click it quickly enough